src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -53,39 +53,87 @@ object DeployPlugin extends AutoPlugin {
 
   object autoImport {
 
-    val deploy = inputKey[Unit](Usage)
+    // The universal packager doesn't clean the staging directory by default.
```

Comment:
You should rephrase as what it's doing - `Cleans the staging directory, which the universal packager doesn't do by default.`